_docs/versioning.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Versioning your Docs
+description: How to turn on and use versioning
+---
+
+# Versioning
+
+Versioning documentation is a useful feature that can be achieved using Docsy Jekyll. It allows you to take snapshots of your documentation that may align with a specific version of a product or service, or you may wish to just keep a historical view of the changes that have been made to your documents over time.
```

Comment:
Let's create a summary after this initial paragraph to quickly explain what adding a version is. E.g., 

To add a new version, you will need to:

1. Copy the _docs content into a subfolder of _docs/Archive (e.g., named by the version)
2. Add a corresponding table of contents to `_data/versions` and to the lookup
3. Test your site locally to ensure it works as expected.

Reply:
Added to the documentation as requested and is part of the latest commit.